primitives/src/communities.rs

```diff
@@ -193,7 +193,7 @@ pub struct CommunityMetadata {
 	/// utf8 encoded abbreviation of the name
 	pub symbol: PalletString,
 	/// ipfs cid for multi-resolution resource for the community icon
```

Comment:
Please also update the doc string.

Reply:
I should be something like:

`IPFS cid to assets necessary for community branding.`